Arista/ConfigFiles/telegraf-qubit-worker.conf

```diff
@@ -20,29 +20,6 @@
   ## Write timeout (for the InfluxDB client), formatted as a string.
   ## If not provided, will default to 5s. 0s means no timeout (not recommended).
   timeout = "10s"
-   namepass = 
```

Comment:
The telegraf host level stats should go to ServerStats database. Thats how the qubit-scylla config is setup.